aip/general/0236.md

```diff
@@ -0,0 +1,361 @@
+---
+id: 236
+state: approved
+created: 2022-03-05
+updated: 2022-03-05
+placement:
+  category: resource-design
+  order: 0
```

Comment:
Not sure what the value of this should be....

Reply:
@toumorokoshi @loudej @shwoodard do you know

Reply:
I'd leave it as zero for now. we can adjust if we find it's in the wrong position.

Reply:
@noahdietz can you remember? I figured it out at one point and then forgot.

Reply:
@nitinapi, just make it 240.